packages/ui/pages/packages/[name].tsx

```diff
@@ -49,34 +49,32 @@ const Package: NextPage = () => {
   return (
     <>
       <h1>Package: {name}</h1>
-      {/* Why aren't the types  */}
       <h3>Versions:</h3>
-      {(data.package as any).variants.map((variant: any) => {
+      {data.package?.variants?.map((variant) => {
```

Comment:
Something for a followup: we need to fix the "Array<_x_ | null?" pattern, this will never happen, the array should always exist and if we have items in it they will be of the type of the array, probably some missing `.nonNull` calls in our type definitions

Reply:
let me make an issue now.

Reply:
https://github.com/es-maintenance/package-inspector/issues/63 issue is created.